ci/unreal-engine.version

```diff
@@ -0,0 +1 @@
+UnrealEngine-something
```

Comment:
Just to point out the obvious, this is a placeholder until the Engine publishing pipeline is fully up and running.  Don't know if we want to make sure to fix this before merging - can probably fix it afterwards if we want to confirm that this is working for a bit before fully relying on it.

Reply:
How are we able to test this currently with this value?

Reply:
@claireimprobable and I manually moved an unreal engine build to that location in the GCS bucket while we were working on the UE build pipeline.